### ISSUE TITLE
Add Kubernetes SubjectAccessReview test

### ISSUE THREAD

**Issue by pehala** [Kuadrant/testsuite#109]
**Summary**
Implement test for Authorization: Kubernetes SubjectAccessReview. Consider rules for access control enforcement based on the Kubernetes authorization system: Role, ClusterRole, RoleBinding, and ClusterRoleBinding. 

**Documentation**
https://github.com/Kuadrant/authorino/blob/main/docs/features.md#kubernetes-subjectaccessreview-authorizationkubernetessubjectaccessreview

**Test cases**
 - [ ] Resource attributes
 - [x] Non-resource attributes

**Comment by jsmolar** [comment]
@martinhesko do you have a conclusion for this?

**Comment by averevki** [comment]
Non-resources attributes test: https://github.com/Kuadrant/testsuite/pull/547
